Client/Application/LeanplumIntegration.swift

```diff
@@ -121,22 +123,33 @@ class LeanPlumClient {
             log.info("LeanplumIntegration - Setting up for Production")
             Leanplum.setAppId(settings.appId, withProductionKey: settings.productionKey)
         }
-
+        
         Leanplum.syncResourcesAsync(true)
-
+        
         let attributes: [AnyHashable: Any] = [
             LPAttributeKey.mailtoIsDefault: mailtoIsDefault(),
             LPAttributeKey.focusInstalled: focusInstalled(),
             LPAttributeKey.klarInstalled: klarInstalled(),
             LPAttributeKey.pocketInstalled: pocketInstalled(),
             LPAttributeKey.signedInSync: profile?.hasAccount() ?? false
         ]
-
+        
         self.setupCustomTemplates()
         
+        // This defines an external Leanplum varible to enable/disable FxA prepush dialogs.
+        // The primary result is having a feature flag controlled by Leanplum, and falling back
+        // to prompting with native push permissions.
+        let useFxAPrePush = LPVar.define("useFxAPrePush", with: false)
+        Leanplum.onVariablesChanged {
+            guard let prePushEnabled = useFxAPrePush?.boolValue() else {
```

Comment:
to simplify this you could do a `self.useFxAPrePush = useFxAPrePush?.boolValue() ?? false`